assets/constants.ts

```diff
@@ -7214,6 +7214,24 @@ export default {
         }
       ]
     },
+    "LEARNER_GROUP_EDITOR": {
+      "ROUTE": "edit-learner-group/:learner_group_id",
```

Comment:
Can you confirm this URL pattern works for both backend and frontend? (In the backend we use `<>` for the params placeholder.)

Reply:
yes it do work, its a placeholder anyways. others follow the same pattern

Reply:
Yes, it looks like it's working. But I'm wondering how `<learner_group_id>` and `:learner_group_id` both work in the backend. Do we do any conversions? I'm not sure, do you see any line of code which can explain this? (Just wanted to make sure we know how it's working!)

Reply:
@DubeySandeep Yes, it is being handled by url-interpolation.service. It checks for the <string> pattern and replaces it with the provided value.

for eg, if we call interpolateUrl with the url ```/userinfo/<username>``` and interpolationValues as username: afzal the function will return /userinfo/afzal

Reply:
I'm not worried about the frontend, but looking at how it's being handled in the backend? All these UR:s are bound to OppiaRoot handler:
https://github.com/oppia/oppia/blob/de8b88488d53ef1c1a8469d4f95fd2c8f1499e09/main.py#L1148-L1160

So how it's reading `:learner_group_id` as placeholder similar to `<learner_group_id>`? Does it support both?

Reply:
It did seem to work with ":" as BLOG_AUTHOR_PROFILE_PAGE also uses this, but I didn't find how it is really being handled.

can you help me with this? i guess im missing somehting.

Reply:
I'll look into it tonight, though this comment should not block the PR, can you create an issue to track this task and we should continue working on getting this PR merged?